release-lockfile.spec.md

```diff
@@ -150,8 +150,12 @@ way.
 
 ## Format
 
-The canonical format for the release lockfile is a JSON document containing a
-single JSON object.  
+The canonical format for the release lockfile JSON document containing a
+single JSON object.  Lockfiles **must** conform to the following serialization rules.
+
+* The document must be tighly packed, meaning no linebreaks or extra whitespace.
+* The *all* keys in all objects must be sorted alphabetically.
```

Comment:
We should probably also add that lockfiles with duplicate keys are invalid.

Reply:
What are "the *all* keys"?